packages/kit/types/internal.d.ts

```diff
@@ -380,7 +367,7 @@ export interface SSRRoute {
 export interface SSRState {
 	fallback?: string;
 	getClientAddress(): string;
-	initiator?: SSRRoute | SSRErrorPage;
+	initiator?: 'GENERIC_ERROR' | (string & {});
```

Comment:
what does `string & {}` mean?

Reply:
It's a trick so that you still get autocompletion for the special string. If you just do `| string` TS would widen the whole type to just `string`

Reply:
🤯